app/lib/frontend/templates.dart

```diff
@@ -191,6 +192,9 @@ class TemplateService {
         devDeps.isNotEmpty;
 
     final Map<String, dynamic> data = {
+      'package': package,
```

Comment:
Tiny nit: We also use 'package' in the dependencies lists. I know it works due to scoping, but maybe use a different name here?

Reply:
I kind of like the compact name and rely on the scope, but renaming can also work. Any suggestion on how they should be different?

Reply:
It's fine as is. I just stumbled across it, and had to double-check how mustache worked.